sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/export/trace/_utils.py

```diff
@@ -174,29 +173,33 @@ def _get_target_and_path_for_http_dependency(
             # if not default port, include port in target
             if server_port != default_port:
                 target = "{}:{}".format(target, server_port)
-        # We only use these values for target if not already populated by peer.service
-        elif not SpanAttributes.PEER_SERVICE in attributes:
-            # Target from http.host
-            if SpanAttributes.HTTP_HOST in attributes:
-                host = attributes[SpanAttributes.HTTP_HOST]
-                try:
-                    # urlparse insists on absolute URLs starting with "//"
-                    # This logic assumes host does not include a "//"
-                    host_name = urlparse("//" + str(host))
-                    # Ignore port from target if default port
-                    if host_name.port == default_port:
-                        target = host_name.hostname
-                    else:
-                        # Else include the whole host as the target
-                        target = str(host)
-                except Exception:  # pylint: disable=broad-except
-                    pass
-            elif parsed_url:
-                # Target from httpUrl
-                if parsed_url.port and parsed_url.port == default_port:
-                    target = parsed_url.hostname
+        # Target from peer.service
```

Comment:
Could you explain this refactor? Does host and parsed URL not need to be set for any spans with Peer Service?